src/containerapp/azext_containerapp/tests/latest/test_containerapp_java_component.py

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+import os
+import time
+import unittest
+
+from azure.cli.command_modules.containerapp._utils import format_location
+from azure.cli.core.azclierror import ValidationError, CLIInternalError
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse, live_only
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
+from msrestazure.tools import parse_resource_id
+
+from azext_containerapp.tests.latest.common import (write_test_file, clean_up_test_file)
+from .common import TEST_LOCATION, STAGE_LOCATION
+from .utils import create_containerapp_env, prepare_containerapp_env_for_app_e2e_tests
+
+
+class ContainerappJavaComponentTests(ScenarioTest):
```

Comment:
Please add test for bind and unbind java component with `--bind` and `--unbind`.
Also, all new added commands need related test cases.

Reply:
done